userspace/libsinsp/test/async_key_value_source.ut.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <async/async_key_value_source.h>
+#include <gtest/gtest.h>
+#include <cstdlib>
+#include <memory>
+
+class test_key_value_source : public libsinsp::async_key_value_source<std::string, uint64_t> {
+public:
+	test_key_value_source(uint64_t delay_sec, uint64_t wait_response_ms) :
+		async_key_value_source<std::string, uint64_t>(wait_response_ms, UINT64_MAX),
+		m_delay_sec(delay_sec)
+	{}
+
+	virtual ~test_key_value_source()
+	{
+		stop();
+	}
+
+	void stopp()
```

Comment:
Is this really necessary?

Reply:
Oh no, it's not. It's leftover from when I was doing addl testing. Let me remove it.